octopus/0.6.3-beta/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM ubuntu:20.04
+
+# Get all apt dependencies
+RUN apt-get -y update
+RUN apt-get -y install software-properties-common
+RUN apt-get install -y --no-install-recommends apt-utils
```

Comment:
Is this copied over and/or necessary to split apt pulls?

Reply:
This is taken directly from octopus and modified to actually run. They don't distribute the docker image unfortunately so that's why I'm building it here.